core/vibes/soul/sections/error/index.tsx

```diff
@@ -1,18 +1,13 @@
 import { Button } from '@/vibes/soul/primitives/button';
 
 interface Props {
-  title?: string;
-  subtitle?: string;
+  title: string;
+  subtitle: string;
   ctaLabel?: string;
   ctaAction?: () => void | Promise<void>;
 }
 
-export function Error({
-  title = 'Something went wrong!',
-  subtitle = 'Please try again or contact our support team for assistance.',
-  ctaLabel = 'Try again',
-  ctaAction,
-}: Props) {
```

Comment:
Why are we removing defaults here?

Reply:
Good catch. I got confused and thought we had set these in our component.